### DISCUSSION TITLE
Configuration "important"

### DISCUSSION THREAD

**Discussion by handplant** [tailwindlabs/tailwindcss#3120] [General]
Is there a way to assign an id and important to "important" in the configuration? I need both! :)

**Comment by adamwathan** [comment]
Hey! There is not, can you explain the situation where you need both and why?

**Reply by handplant** [reply]
Hey! I have a Vue app which is integrated into various third party websites. I want to protect my CSS from their CSS.

**Reply by handplant** [reply]
Ok, my mistake! :) ID is quite enough.

But another thing. is it possible to integrate tailwindcss/base into the ID wrapper?
